rbac/management/principal/view.py

```diff
@@ -116,7 +119,9 @@ def get(self, request):
                 user.account, email=email, limit=limit, offset=offset, sort_order=sort_order
             )
         else:
-            resp = proxy.request_principals(user.account, limit=limit, offset=offset, sort_order=sort_order)
+            resp = proxy.request_principals(
```

Comment:
Just looking at this and #342 and curious if at some point we might want to make an effort to refactor some of this to not cause the signature of this to continue to grow. Just wanted to point that out since I'm sure you'll have to rebase one of these. :)

Reply:
👍 I will merge the other one and rebase this one to refactor it